v03_pipeline/lib/misc/family_loading_failures.py

```diff
@@ -175,10 +178,18 @@ def get_families_failed_sex_check(
     failed_families = defaultdict(list)
     for family in families:
         for sample_id in family.samples:
-            if family.samples[sample_id].sex not in {
-                sex_check_lookup[sample_id],
-                Sex.UNKNOWN,
-            }:  # NB: Unknown samples in pedigree are excluded from sex check.
+            # NB: Both Unknown samples in pedigree and Unknown
+            # samples in the predicted_sex are precluded from
+            # failing the sex check.
+            if (
+                sex_check_lookup[sample_id] == Sex.UNKNOWN  # noqa: PLR1714
+                or family.samples[sample_id].sex == Sex.UNKNOWN
+            ):
+                logger.info(
+                    f'Encountered sample with Unknown sex excluded from sex check: {sample_id}'
```

Comment:
and these are gonna be strings, right, not a Hail string type thtat's goint to print as <Hail.String type at 07x40194578> or whatever , yea ? If so then lgtm

Reply:
Ya, everything is `hl.eval`ed by this point.  The [signature of the](https://github.com/broadinstitute/seqr-loading-pipelines/pull/900/files/bb440635cbdfcad1e0114b9b70803e81a0553069#diff-81bda30e9edc96dbaf6b9533d4db17f8afa17df35de4b37c3c05e1856e6779dbL125) `build_sex_check_lookup` is correct I think!